extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/handler/NtpHandler.java

```diff
@@ -204,8 +207,9 @@ private synchronized void refreshTimeDate() {
 
             Calendar calendar = Calendar.getInstance(timeZone, locale);
             calendar.setTimeInMillis(networkTimeInMillis);
+            ZonedDateTime zoned= ZonedDateTime.of(LocalDateTime.now(),timeZone.toZoneId());

```

Comment:
You can drop the above calendar instance now, can't you?

Reply:
Having a closer look I am afraid that it would make troubles because it is used to update the StringType channel. Since I am changing only the DateTimeType impacts I decided to keep it like this. Would this be okay for you?